include/unifex/any_unique.hpp

```diff
@@ -492,12 +210,6 @@ using any_unique = typename _any_unique::_byval<CPOs...>::type;
 template <auto&... CPOs>
 using any_unique_t = any_unique<tag_t<CPOs>...>;
 
-template <typename... CPOs>
-using any_ref = typename _any_unique::_byref<CPOs...>::type;
-
-template <auto&... CPOs>
-using any_ref_t = any_ref<tag_t<CPOs>...>;
-
 } // namespace unifex
 
 #include <unifex/detail/epilogue.hpp>
```

Comment:
Is it not possible to implement `any_unique` in terms of `basic_any_object`?

Reply:
I guess in theory, `any_unique` could be implemented in terms of `basic_any_object` with an inline-size of 0.

There might be some small perf difference, though, in that with any_unique we know that the data pointer is always a pointer and so the caller of a CPO can load the data pointer in a register and pass that directly (possibly avoiding needing to spill the any_unique object to the stack). Whereas with any_object, since the object might be stored inline we have to pass the address of the data pointer into the CPO function which then interprets that memory as a pointer and loads the address of the object. This seems more likely to require the compiler to write the type-erasing wrapper to memory.

This is all just a hunch, though. I've not benchmarked or measured the runtime performance difference here....